File: Day_26/Q102.java
import java.util.*;

class Q102{
    static Scanner sc = new Scanner(System.in);

    //Check Eligibility
    static boolean is_eligible(int age){
        return age >= 18;
    }

    public static void main(String[] args){
        int age = sc.nextInt();

        if(age < 0){
            System.out.println("Invalid Age");
        }
        else if(is_eligible(age)){
            System.out.println("Eligible To Vote");
        }
        else{
            System.out.println("Not Eligible For Voting");
        }

        sc.close();
    }
}

File: Day_26/Q102.cpp
#include<bits/stdc++.h>
using namespace std;

//Check Eligibility
bool is_eligible(int age){
    return age >= 18;
}

int main(){
    int age;
    cin >> age;

    if(age < 0){
        cout << "Invalid Age";
    }
    else if(is_eligible(age)){
        cout << "Eligible To Vote";
    }
    else{
        cout << "Not Eligible For Voting";
    }

    return 0;
}
